xrpl/utils/txn_parser/utils/balance_parser.py

```diff
@@ -105,16 +104,16 @@ def _get_trustline_quantity(
     return []
 
 
-def _group_balance_changes(
-    balance_changes: List[BalanceChange],
-) -> Dict[str, List[BalanceChange]]:
-    grouped_balance_changes: Dict[str, List[BalanceChange]] = {}
-    for change in balance_changes:
-        account = change["account"]
-        if account not in grouped_balance_changes:
-            grouped_balance_changes[account] = []
-        grouped_balance_changes[account].append(change)
-    return grouped_balance_changes
+def _group_balance(
```

Comment:
```suggestion
def _group_balances(
```

This applies to multiple balances :)